### PR TITLE
Update readme with error handling example

### PR DESCRIPTION
**Pull request by sosaucily**
I struggled for a little too long to figure out how to best catch with error from a param validation failure. It was causing my server to 500, even in production. Yuck city. Thought this example would be helpful. Thanks for the library!

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1156,6 +1156,20 @@ is raised and can be rescued and processed. It contains a description
 of the parameter value expectations. Validations can be turned off
 in the configuration file.
 
+Here is an example of how to rescue and process a +ParamMissing+ or
++ParamInvalid+ error from within the ApplicationController.
+
+.. code:: ruby
+  class ApplicationController < ActionController::Base
```
Comment (mathieujobin) [inline README.rst:1163]: this code isn't displayed then the README is rendered.
I think you are only missing a white line after `code:: ruby`
can you try adding one?

**Review by mathieujobin — COMMENTED** [review]
Thanks for the update, this is useful. please address the one comment I left and I will merge

File: README.rst (hunk 1 above)
Comment (sosaucily) [reply README.rst:1164]: Good catch. Fixed! Looks better now

**Comment by sosaucily** [comment]
Good catch @mathieujobin Fixed now!

**Review by mathieujobin — APPROVED** [review]

**Pull request merged**